### ISSUE TITLE
Duplication of compendium with folders loses folder structure

### ISSUE THREAD

**Issue by toastygm** [foundryvtt/foundryvtt#9333]
### What happened?

I created an Items compendium pack with a folder structure and placed all the items in the compendium pack into folders.  I then duplicated the compendium, and all of the items got duplicated but the whole folder structure is gone in the duplicated compendium.

I expected the duplicated compendium to have all of the items in the folders in the same way as in the original compendium.

### What ways of accessing Foundry can you encounter this issue in?

- [X] Native App (Electron)
- [X] Chrome
- [X] Firefox
- [X] Safari
- [ ] Other

### Reproduction Steps

# Create a new compendium
# Create a new folder in the compendium
# Create a new item in the folder in the compendium
# Duplicate the compendium

The duplicated compendium will have the item, but not the folder structure from the source compendium.

### What core version are you reporting this for?

Version 11 Build 296

### Bug Checklist

- [X] The issue occurs while all Modules are disabled

**Comment by aaclayton** [comment]
Makes sense because the folder mapping is stored separately. We'll need to extend the compendium duplication workflow to also copy any folder mapping that exists for the duplicated pack.
